automated install/basic-install.sh

```diff
@@ -1772,9 +1772,22 @@ finalExports() {
     echo "INSTALL_WEB_INTERFACE=${INSTALL_WEB_INTERFACE}"
     echo "LIGHTTPD_ENABLED=${LIGHTTPD_ENABLED}"
     echo "CACHE_SIZE=${CACHE_SIZE}"
-    echo "DNS_FQDN_REQUIRED=true"
-    echo "DNS_BOGUS_PRIV=true"
     }>> "${setupVars}"
+
+    # if this runs the first time (variable not set yet) set the varibale to true otherwiese keep the old value
+    if [[ ! -v DNS_FQDN_REQUIRED ]]; then
+      echo "DNS_FQDN_REQUIRED=true" >> "${setupVars}"
+    else
+      echo "DNS_FQDN_REQUIRED=${DNS_FQDN_REQUIRED}" >> "${setupVars}"
+    fi
```

Comment:
```suggestion
    echo "DNS_FQDN_REQUIRED=${DNS_FQDN_REQUIRED:-true}" >> "${setupVars}"
```

Reply:
From the `bash` documentation:
``` bash
${parameter:-word}
#    If parameter is unset or null, the expansion of word is substituted. 
#    Otherwise, the value of parameter is substituted.
```

Example (`a` is not set initially):
``` bash
$ echo "$a"

$ echo "${a:-default}"
default

$ a="abc"
$ echo "${a:-default}"
abc
```